### ISSUE TITLE
How to add calendar widget?

### ISSUE THREAD

**Issue by jbhanks** [zadam/trilium#909]
I am trying to figure out how to embed the javascript calendar widget in trilium note so that I can use it. I found the calendar widget at https://github.com/zadam/trilium/blob/master/src/public/javascripts/widgets/calendar.js  

When I create a JS frontend (or backend) note, an `x` appears before the first line saying `Parsing error: 'import' and 'export' may appear only with 'sourceType: module`. 

What do I need to do to get this widget working?

* `/home/[user]/.local/share/trilium-data/log` for Linux
* `C:\Users\[user]\AppData\Roaming\trilium-data\log` for Windows Vista and up
* `/Users/[user]/Library/Application Support/trilium-data/log` for Mac OS

**Comment by zadam** [comment]
Calendar widget automatically appears on day notes? Where do you want to put it instead?

**Comment by jbhanks** [comment]
I was trying to embed a calendar in a note itself so that it would be bigger, but the way it is works fine. Is there any way to have the calendar itself show items that are entered for days (or at least highlight days that have entries?)

**Comment by zadam** [comment]
> Is there any way to have the calendar itself show items that are entered for days (or at least highlight days that have entries?)

No, calendar widget is currently pretty simplistic. It would be possible to create a customer widget though with some scripting ...

**Comment by jbhanks** [comment]
If I did that, could I create new calendars in new notes? Actually, I'm not sure how to create new calendars with the existing widget, it would be nice to have a couple of separate calendars for different things.
